students/BrianB/lesson02/grid_printer.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+
+#--------------------------------------------------
+# Who: Brian Brumleve
+# What: Lesson02 - Grid Printer
+# When: April 21, 2019
+# Program Description:
+#   This program uses a input parameters to print
+#   out two grids.  One grid, my_grid1() prints a
+#   grid that grows/shrinks based on a single
+#   input parameter.  The second grid, my_grid2()
+#   prints a grid that grows/shrinks in the # of
+#   columns and rows as well grows/shrinks the
+#   width of those columns and rows based on
+#   two input parameters.
+#--------------------------------------------------
+
+
+# The row() function creates
+# the tops and bottoms of the
+# grids as the for-loop determines
+# how wide each space between
+# pluses "+" will be, as shown
+# by the dash " - " lines.
+def row(b, a=2):
+    # The 1st part of this function,
+    # normalizes the input, b, by
+    # establishing the range is an
+    # integer and ensuring integration
+    # with the col() function to
+    # ultimately print the correct
+    # width rows: plus signs and dashes
+    # and the correct width columns:
+    # vertical lines and spaces.
+    if b % 2 == 0:
+        x = int(b) / 2
+    elif b % 2 == 1:
```

Comment:
same as before, no need for a elif if it's always going to be True.

you could also use // for "floor division", rather than checking for even or odd:

x = b // 2

```
In [17]: for i in range(8): 
    ...:     print(i // 2) 
    ...:                                                                        
0
0
1
1
2
2
3
3
```